components/script/dom/webglrenderingcontext.rs

```diff
@@ -2235,7 +2238,7 @@ impl WebGLRenderingContextMethods for WebGLRenderingContext {
                     WebGLFramebufferBindingRequest::Default
                 ))
             );
-            framebuffer.delete(false)
+            framebuffer.delete(false);
```

Comment:
Does this give us a warning about an ignored result now?

Reply:
That change isn't necessary. (but no is the answer)